### ISSUE TITLE
[4.0] Font differences betwen editor and site

### ISSUE THREAD

**Issue by bato3** [joomla/joomla-cms#30336]
### Steps to reproduce the issue
Create new content: article. (editor: TinyMce)

Inspect paragraph

### Expected result

```css
box-sizing: border-box;
color: rgb(33, 37, 41);
font-family: "Fira Sans", Arial, sans-serif;
font-size: (body: 1rem, html 14px) 14px;
font-weight: 400;
line-height: (body: 1.6)  22.4px;
margin-bottom: 14px;
margin-top: 0px;
text-align: left
```

### Actual result
```css
color: rgb(51, 51, 51);
font-family: Tahoma, Helvetica, Arial, sans-serif;
font-size: (body: 12px) 12px;
line-height: (body: 1.3em) 15.6px;
overflow-wrap: break-word
```

### System information (as much as possible)
Clean installation, debian 10,  default configuration
release Joomla 4.0 Beta 3
screen size: 1920 x 1080, editor: TinyMce

### Additional comments
It would be nice if the page template included an editor style generated from the same settings.
It would be nice if the editor width was close to the width of the page content.

**Comment by chmst** [comment]
Thank you very much for your hint. We are working in this area and will take this on our list. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30336">issues.joomla.org/tracker/joomla-cms/30336</a>.</sub>

**Comment by Quy** [comment]
Related #29031

**Comment by bato3** [comment]
I suggest create `editor.scss` in cassopea template with content:
```scss

/*TYPOGRAPHY FOR EDITOR**/

html {
  background-color: $white; // I can't find special variable for content background. In cassiopea/scss/blocks/global it's forced to  #fff
  padding:7px;
  margin:0;
}

@import "../../../media/vendor/bootstrap/scss/functions";

@import "variables";

@import "mixin";

// Bootstrap
@import "../../../media/vendor/bootstrap/scss/variables";
@import "../../../media/vendor/bootstrap/scss/functions";
@import "../../../media/vendor/bootstrap/scss/mixins";
@import "../../../media/vendor/bootstrap/scss/type";
@import "blocks/global"; // Leave this first
@import "vendor/bootstrap/table";


/* STYLES FOR JOOMLA! EDITOR */
hr#system-readmore  { border: red dashed 1px; color: red; }
hr.system-pagebreak { border: gray dashed 1px; color: gray; }


/* Classes for style picker */

.float-left  { float: left !important; }
.float-right { float: right !important; }

table {
  &.table, &.table-striped, &.table-hover {
    width:100%; // fake content, to don't remove empty rules, it's OK for me
  }
  &.table-sm {
    width:100%; // fake content, it's OK for me
    th,
    td {
      padding: $table-cell-padding-sm;
    }
  }
  &.table-bordered {
    border: $table-border-width solid $table-border-color;
    th,
    td {
      border: $table-border-width solid $table-border-color;
    }
  }
}
tr.thead-dark {
  width:100%; // fake content, 
  th {
    color: $table-dark-color;
    background-color: $table-dark-bg;
    border-color: $table-dark-border-color;
  }
}

tr.thead-light {
  width:100%; // fake content, 
  th {
    color: $table-head-color;
    background-color: $table-head-bg;
    border-color: $table-border-color;
  }
}

```
I put it so ugly, because I'm not quite able to work with pull-requests, especially since these changes need to be added to 4.0, 4.1, 4.2 and applied to the compiling library...

**Comment by HLeithner** [comment]
closed because of PR #31828
